nestjs-prisma/.env.example

```diff
@@ -0,0 +1,9 @@
+# Environment variables declared in this file are automatically made available to Prisma and Nx.
+# Prisma Details: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+# Nx Details: https://nx.dev/recipes/environment-variables/define-environment-variables
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_ONE_URL="file:./dev.db"
+DATABASE_TWO_URL="file:./dev.db"
```

Comment:
These shouldn't point to the same file path right?

Reply:
the path will be relative to where the schema imports it. so 
`libs/prisma-schema-one/prisma/schema.prisma` => `libs/prisma-schema-one/prisma/dev.db`
`libs/prisma-schema-two/prisma/schema.prisma` => `libs/prisma-schema-two/prisma/dev.db`

When using not sqlite they'd be different connection strings so it would make more sense when looking at it. 
Probably best to comment as to why those are both the same here to prevent confusion.